### DISCUSSION TITLE
Proposal

### DISCUSSION THREAD

**Discussion by markrieder** [spacedriveapp/spacedrive#2279] [Feature Request]
1. Organization of files and folders by date, name...
2. Different cloud providers under one roof https://github.com/rclone/rclone
3. Useful function of merging disks or virtual disks into a single point of view https://github.com/trapexit/mergerfs

**Comment by utkubakir** [comment]
You can already sort by date, name etc and 2 is coming soon. 3 sounds nice, not sure if it would be a priority though.
